backend/django_core/static_src/components/page/explore/Index.svelte

```diff
@@ -0,0 +1,220 @@
+                            type="text"
+                            tabindex="0" role="button"
+                            class="peer w-full rounded-lg border-none bg-neutral py-3 text-base leading-none placeholder focus:ring-0 md:w-[11vw] md:rounded-[0.5vw] md:bg-neutral text-neutral-content md:py-[0.8vw] md:pl-[1vw] md:text-[1vw] placeholder:font-medium font-semibold"
+                        />
+                        {#if selected_items.length > 0}
+                            <button
+                                on:click={() => clear_selected_items(option[0])}
+                                class="btn !bg-transparent border-none absolute right-0 mr-3 w-4 p-0 md:mr-[1vw] md:w-[1vw]"
+                            >
+                                <coreproject-icon-cross class="md:w-[1vw]"></coreproject-icon-cross>
+                            </button>
+                        {:else}
+                            <button class="btn !bg-transparent border-none absolute right-0 mr-3 w-4 p-0 md:mr-[1vw] md:w-[1vw]">
+                                <coreproject-icon-chevron class="md:w-[1vw]"></coreproject-icon-chevron>
+                            </button>
+                        {/if}
+                    </div>
+                    <!-- svelte-ignore a11y-no-noninteractive-tabindex -->
+                    <div
+                        tabindex="0"
+                        class="dropdown-content z-10 md:mt-[1vw] w-[8.5rem] rounded-lg md:w-[11vw] md:rounded-[0.5vw] overflow-x-hidden"
+                    >
+                        {#if filter_items}
+                            <ScrollArea
+                                class="md:p-[0.35vw] flex flex-col w-full"
+                                parent_class="md:max-h-[30vw] bg-neutral w-full"
+                            >
+                                {#each Object.entries(filter_items) as item}
+                                    {@const key = item[0]}
+                                    {@const value = item[1]}
+
+                                    {@const is_selected = selected_items.some(selected_item => selected_item[0] === key)}
+
+                                    <button
+                                        on:click={() => update_selected_items(option[0], item)}
+                                        class="relative btn btn-neutral h-max min-h-max leading-none flex items-center justify-start md:rounded-[0.35vw] p-3 text-sm md:px-[1vw] md:py-[0.75vw] md:text-[0.9vw]">
+                                        {value}
+
+                                        {#if is_selected}
+                                            <div class="absolute right-[0.75vw] rounded-full bg-primary text-white p-1 md:p-[0.25vw]">
+                                                <Tick class="w-2 md:w-[0.75vw] text-white" />
+                                            </div>
+                                        {/if}
```

Comment:
maybe add animation

Reply:
For what? this `.dropdown` has already a nice animation